heatmap.py

```diff
@@ -658,14 +662,13 @@ def create_movie(infiles, outfile, ffmpegopts):
         logging.info('Encoding video: %s' % ' '.join(command))
         subprocess.call(command)
 
-
     def run(self):
         logging.info('Putting animation frames in %s' % self.tmpdir)
         self.inputs_since_output = 0
         self.frame_count = 0
         matrix = process_shapes(self.config, self.maybe_save_image)
-        if ( not self.frame_count
-             or self.inputs_since_output >= self.config.straggler_threshold ):
+        if (not self.frame_count or
+                self.inputs_since_output >= self.config.straggler_threshold):
```

Comment:
If it was, it reports:
`heatmap.py:671:13: E129 visually indented line with same indent as next logical line`